scripts/crowbar-testbuild.yaml

```diff
@@ -11,6 +11,10 @@ branches:
     bs_project:  Devel:Cloud:7:Staging
     bs_repo:     SLE_12_SP2
     cloudsource: develcloud7
+  master:
```

Comment:
are you sure that we can have more than one "master" key in this dict? that sounds strange. it would be more logical to me to have the master hash from above (line 11) changed to cloud8, and a stable/4.0 added in addition (it doesn't harm to do that right away, because it won't be used unless there is a stable/4.0 branch)